docs/sql-keywords.md

```diff
@@ -103,6 +103,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>DROP</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>ELSE</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>END</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
+  <tr><td>ESCAPE</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
```

Comment:
ESCAPED is reserved in the standard: https://developer.mimer.com/wp-content/uploads/standard-sql-reserved-words-summary.pdf

Reply:
Thanks for your reminder, I will change it.